docs/components/modeler/web-modeler/additional-features/call-activity-linking.md

```diff
@@ -0,0 +1,27 @@
+---
+id: call-activity-linking
+title: Call activity linking
+description: Call activity linking
+---
+
+You can use one of the following approaches to link a [call activity](/components/modeler/bpmn/call-activities/call-activities.md) to another process.
```

Comment:
Please explain why the user should do that. What are the benefits of call activities? Why do I need to link the processes? Is this adding convenience (i.e. I do not need to enter process IDs)?
You may just write a short sentence why call activities are helpful (... to reduce complexity and support reusability, you can call another process as a building block using call activities... if the child process model is part of the project of your parent model, you can link them using following approaches...) and then reference the call activity doc as you did (https://docs.camunda.io/docs/components/modeler/bpmn/call-activities/)

Reply:
(I also think the sentence is semantically wrong: you don't link the call activity to another process, but you link a child process via a call activity to another process.)

Reply:
In that case, does it make sense to call this feature Call activity linking?

Also, should this section be used to describe the benefits of call activities? Shouldn't it be in the call activity doc (like it is now) and shouldn't we use this section to explain how to create the link?

Reply:
I changed the text to:
_You can use one of the following approaches to link the process to be called by a call activity_.

Is this better?

Reply:
I was also confused by this. Perhaps "Use one of the following approaches to link one process to another using a call activity."?